sections/admin/clients/new-client.jsx

```diff
@@ -16,8 +17,33 @@ const AdminNewClient = () => {
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    toast.success("Успішно");
-    setTimeout(() => Router.push("/admin/clients"), 500);
+    const dataNewClient = { name, carBrand, phoneNumber, carModel, email, licensePlate };
+
+    const pattern = /^[a-z]+\s[a-z]+$/i;
```

Comment:
This pattern does not allow write name on Ukrainian or just a name (without surname)
I would not like to limit user in naming a client. It must be ability to name whatever you want.
For instanse: "Гнат", "Ляшенко Г.О.", "Гнат Л.", "Hnat", "Гнат!?", "Гнат$$" whatever..

Reply:
It is better to just check if this field empty or not

Reply:
As I said I disagree because it's not nick or others if it's "Name and Surname" that it's "Name and Surname".
But I'm out of business logic here, so, I'll do as u say.